modules/con-vuln-sources.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * architecture/acs-architecture.adoc
+// * cloud_service/acscs-architecture.adoc
+:_mod-docs-content-type: CONCEPT
+[id="con-vuln-sources_{context}"]
+= Vulnerability data sources
+
+Sources for vulnerabilities depend on the scanner type and {product-title-short} version used. {product-title-short} contains two scanners: the StackRox Scanner and Scanner V4.
+
+The StackRox Scanner originates from a fork of the Clair v2 open source scanner and is the default scanner. The StackRox Scanner is deprecated beginning with release 4.6. In version 4.4, {product-title-short} introduced Scanner V4, built on ClairCore, which provides additional image scanning features.
+
+[id="stackrox-scanner-vuln-sources"]
+== StackRox Scanner sources
+
+The StackRox Scanner uses the following vulnerability sources:
+
+* https://access.redhat.com/security/data/oval/v2/[Red{nbsp}Hat OVAL] v2
+* link:https://secdb.alpinelinux.org/[Alpine Security Database]
+* Data tracked in link:https://alas.aws.amazon.com/index.html[Amazon Linux Security Center]
+* link:https://security-tracker.debian.org/tracker/data/json[Debian Security Tracker]
+* link:https://linux.oracle.com/security/oval[Oracle OVAL]
+* link:https://packages.vmware.com/photon/photon_oval_definitions/[Photon OVAL]
+* link:https://support.novell.com/security/oval/[SUSE OVAL]
+* link:https://security-metadata.canonical.com/oval/[Ubuntu OVAL]
+* link:https://nvd.nist.gov/vuln/search[NVD]: This is used for various purposes such as filling in information gaps when vendors do not provide information. For example, Alpine does not provide a description, CVSS score, severity, or published date.
++
+[NOTE]
+====
+This product uses the NVD API but is not endorsed or certified by the NVD.
+====
+* link:https://github.com/stackrox/stackrox/blob/master/scanner/updater/manual/vulns.go[StackRox]: The upstream StackRox project maintains a set of vulnerabilities that might not be discovered due to data formatting from other sources or absence of data.
```

Comment:
For Scanner v2, it's these files, instead:

https://github.com/stackrox/scanner/blob/master/ext/vulnsrc/manual/manual.go

https://github.com/stackrox/scanner/blob/master/pkg/vulnloader/nvdloader/manual.go